ingest/user_plugins/test_plugin.py

```diff
@@ -1,6 +1,39 @@
 from plugins.file_data_source import FileDataSource
+from time import sleep
+import logging
+
+log = logging.getLogger(__name__)
 
 
 class Plugin(FileDataSource):
-    def print(self):
-        print("test 2")
+    def __init__(self, runtime_id, name):
+        # TODO: somehow, we need to pass our three overidden methods back to the base class
+        # probably could be some clever kwargs thing, but am too tired to solve it.
+        super().__init__(runtime_id, name)
```

Comment:
@mmontgomery95 this is our big riddle for today...how can we let the base class constructor know about the subclass's three overrides?